seqr/views/apis/project_api.py

```diff
@@ -215,6 +218,7 @@ def project_families(request, project_guid):
         families_by_id[family_id].update({
             'individualGuids': sorted(id_guid_map.values()),
             'hasPhenotypePrioritization': family_id in phenotype_priority_families,
+            'hasRna': family_id in rna_families,
```

Comment:
the RNA samples on the project page are only used for filtering the family table, the detailed models are never used themselves, so theres no need to return all those models as long as we add this extra aggregation